src/rgw/driver/rados/rgw_rest_user.cc

```diff
@@ -19,6 +19,23 @@
 
 using namespace std;
 
+int fetch_access_keys_from_master(const DoutPrefixProvider *dpp, rgw::sal::Driver* driver, RGWUserAdminOpState &op_state, req_state *s, optional_yield y) {
+    bufferlist data;
+    JSONParser jp;
+    RGWUserInfo ui;
+    int op_ret = driver->forward_request_to_master(s, s->user.get(), nullptr, data, &jp, s->info, y);
+    if (op_ret < 0) {
+      ldpp_dout(dpp, 0) << "forward_request_to_master returned ret=" << op_ret << dendl;
+      return op_ret;
+    }
+    ui.decode_json(&jp);
+    op_state.op_access_keys = ui.access_keys;
```

Comment:
this assignment is a good place to use move semantics. `RGWUserInfo ui` is about to go out of scope, so we can move it's access keys instead of copying them:
```c++
op_access_keys = std::move(ui.access_keys);
```